source/operations/network-encryption.rst

```diff
@@ -15,23 +15,34 @@ MinIO supports Transport Layer Security (TLS) 1.2+ encryption of incoming and ou
 .. admonition:: SSL is Deprecated
    :class: note
 
-   TLS is the successor to Secure Socket Layer (SSL) encryption. 
+   TLS is the successor to Secure Socket Layer (SSL) encryption.
    SSL is fully `deprecated <https://tools.ietf.org/html/rfc7568>`__ as of June 30th, 2018.
 
 .. _minio-tls-user-generated:
 
 Enabling TLS
 ------------
 
+.. cond:: not k8s
+
+   The sections below describe how to enable TLS for MinIO.
+   You may use TLS certificates from a well-known Certificate Authority, an internal or private CA, or self-signed certs.
+
+   Before beginning, note these important points:
+
+   - Configure TLS on each node.
+   - Ensure certs are readable by the user who runs the MinIO Server process.
+   - Update ``MINIO_VOLUMES`` and any needed services or apps to use an ``HTTPS`` URL.
```

Comment:
Should this be a link to the envvar?

```suggestion
   - Update :envvar:`MINIO_VOLUMES` and any needed services or apps to use an ``HTTPS`` URL.
```

Reply:
@djwfyi oh yes, I need to add that. Thank you for the reminder! I did something wrong when I tried, and hadn't gotten back to it.